tests/foreman/api/test_template.py

```diff
@@ -305,3 +305,334 @@ def test_positive_clone(self):
                 for key in uniqe_keys:
```

Comment:
@cswiii yes I agree but the code above do not belong to this PR, we can fix it in a different PR maybe.